### PR TITLE
Add archival script

### PR DESCRIPTION
**Pull request by schencej**

### CODE REVIEW

File: docs/archive/scrape_contribs.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                print("Found demo, navigation to URL...")
+                url = resc['url']
+                try:
+                    browser.get(url)
+                except:
+                    print(f"Navigating to demo url failed: {url}")
+                input("Press enter to continue...")
+            if resc_type == 'data':
+                print("Skipping data")
+                continue
```
Comment (brianhhu) [inline docs/archive/scrape_contribs.py]: Minor nitpick: can you add a newline here?

File: docs/archive/README.txt [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+     - software is a link to a zip file that doesn't seem to exist, I think this might require a science.org account as well
+
+    bayesian_teaching_rutgers:
+     - 5th paper is behind SPIE paywall
+
+    utdallas-ufl-xar-demo:
+     - demo is a web app; not archived
+
+    utd-ucla-juice:
+     - software points to a group of github repos; archived the main repo
```
Comment (brianhhu) [inline docs/archive/README.txt]: Minor nitpick: can you add a newline here?

File: docs/archive/README.txt [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+The script uses a Firefox browser currently.
+A new directory "contribs" is created with a sub-directory for each contribution.
+
+For papers, the script navigates to the associated URL and waits for user input to continue.
+You should download the paper and place it in the folder created for the contribution.
+
+For software, the script attempts to clone and bundle it as a git repo and place the bundle file in the contrib directory.
+If this fails, it navigates to the URL for the user to figure out.
+Repos are all cloned into their own sub-directory of "repos".
+The "repos" directory can be deleted after the archive is complete.
```
Comment (brianhhu) [inline docs/archive/README.txt:19]: I think we should add one line or a note here about how to convert git bundles into actual usable code. Maybe something like, "To restore the original repos, you must have `git` installed. You can then simply use `git clone` on each bundle. For more information about git bundling, please view the related [documentation](https://git-scm.com/book/en/v2/Git-Tools-Bundling)."

**Review by brianhhu — APPROVED** [review]
LGTM, left a few very minor comments. Thanks!

**Comment by schencej** [comment]
Amended commit with fixes.

**Comment by brianhhu** [comment]
LGTM, thanks!

**Pull request merged**